src/main/java/com/android/volley/toolbox/HurlStack.java

```diff
@@ -153,10 +154,10 @@ private static boolean hasResponseBody(int requestMethod, int responseCode) {
      * Wrapper for a {@link HttpURLConnection}'s InputStream which disconnects the connection on
      * stream close.
      */
-    static class UrlConnectionInputStream extends FilterInputStream {
+    protected static class UrlConnectionInputStream extends FilterInputStream {
```

Comment:
I would rather not expose this implementation detail as part of Volley's API. I don't think you should need it - instead of extending it, you can build your own FilterInputStream subclass which filters the result of super.createInputStream() using composition rather than inheritance.